thoth/package_extract/image.py

```diff
@@ -546,6 +546,50 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    # gathering version from version.txt file
+    version_path = path + "/usr/local/cuda/version.txt"
+    if os.path.isfile(version_path):
+        with open(version_path, "r") as f:
+            for line in f.readlines():
+                if line.startswith("CUDA Version"):
+                    res["/usr/local/cuda/version.txt"] = line[len("CUDA Version"):].strip()
+                    _LOGGER.info("CUDA version identifier was found in file %s", res["/usr/local/cuda/version.txt"])
+                    break
+                else:
+                    _LOGGER.debug("No CUDA version identifier was found in file %r", version_path[len(path):])
+    else:
+        _LOGGER.warning("No %r file was found", version_path[len(path):])
+
+    # gathering version from nvcc command
+    nvcc_path = glob.glob(path + '/usr/local/cuda/bin/nvcc')
+    if len(nvcc_path) > 0:
+        st = os.stat(nvcc_path[0])
+        os.chmod(nvcc_path[0], st.st_mode | stat.S_IEXEC)
+        result = run_command("{} --version".format(nvcc_path[0]), raise_on_error=False)
+        if result.return_code != 0:
+            _LOGGER.warning(
+                "Unable to detect CUDA version - nvcc returned non-zero version: %s",
+                result.to_dict(),
+            )
+        else:
+            for line in result.stdout.splitlines():
+                if line.startswith("Cuda compilation tools"):
+                    for version in line.split(", "):
+                        if version.startswith("V"):
```

Comment:
> https://github.com/thoth-station/package-extract/pull/307#discussion_r356041025
Now I first try to find a line which starts with `Cuda compilation tools`, then take that line and split it by `, ` and look for string which starts with `V`. As a reminder on the output of this command result:
```
root@c3624a58d7d4:/usr/local# nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2019 NVIDIA Corporation
Built on Sun_Jul_28_19:07:16_PDT_2019
Cuda compilation tools, release 10.1, V10.1.243
```